tutorials/Linear Combination of Unitaries/LCU Tutorial .ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>Please add here a GIF / figure of the entire circuit as viewed in the IDE </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>